app/jobs/detect_sidekiq_metrics_issues_job.rb

```diff
@@ -3,7 +3,7 @@
 class DetectSidekiqMetricsIssuesJob
   include Sidekiq::Worker
 
-  SIDEKIQ_QUEUE_NAMES = %w[priority_mailers mailers default participant_outcomes big_query].freeze
```

Comment:
maybe we can remove outcomes from fixtures? `fixtures/files/api_reference/api_spec.json`